src/scenes/AcceleratorRouter/Deliverables/ApproveDeliverableDialog.tsx

```diff
@@ -17,6 +21,22 @@ export default function ApproveDeliverableDialog({
   approveMessage,
 }: ApproveDialogProps): JSX.Element {
   const theme = useTheme();
+  const { activeLocale } = useLocalization();
+
+  const defaultApproveMessages = useMemo(() => {
+    if (!activeLocale) {
+      return undefined;
+    }
+
+    switch (deliverableType) {
```

Comment:
One of the changes made here is to use different messages for approving questions deliverables. Currently it shows a message about approving species deliverable which seems inaccurate.